docs/contributing.rst

```diff
@@ -211,12 +212,6 @@ Some details about using Modin on Ray with Asv:
   a large overhead, since for each process redis server and other necessary binaries
   from ray initialization are started and destroyed.
 
-Some details for maintenance:
-
-* ``modin/asv_bench/asv.conf.json`` contains the modin dependencies, with which testing takes place.
-
-We need to keep them up to date according to the dependencies in ``setup.py``.
-
```

Comment:
Now this is not the case; dependencies are determined automatically by `setup.py`.